src/bin/server.cpp

```diff
@@ -15,7 +16,10 @@ int main(int argc, char* argv[]) {
     uint16_t port = 5432;
 
     if (argc >= 2) {
-      port = static_cast<uint16_t>(std::atoi(argv[1]));
+      char* endptr{nullptr};
```

Comment:
`type_cast`?

Reply:
I think this version has a more robust error handling

Reply:
This version has unintended behaviour if the specified port is `>= 2 ^ 16` or less than zero ;) Don't know if type_cast (aka boost literal_cast) handles this better, though.

Reply:
Nope, doesn't. Will fix.